payments/templates/payments/order_payment.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+{% load custom_filters %}
+
+{% block title %}
+  {% trans "Order Detail" %}
+{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="text-center">
+          <h2>{% trans "Pay the order" %} #{{ order.variable_symbol }}</h2>
+        </div>
+        <hr>
+        <div class="row">
+          {# todo: render is separately in future and display it only when it responses to signal #}
+          <div class="col-md-12">
+            <h3>{% trans "Pay by bank transfer" %}</h3>
+            <div class="alert alert-success" role="alert">
+              {% blocktrans with amount=order.left_to_pay|currency variable_symbol=order.variable_symbol %}
+                Transfer {{ amount }} to the account bellow and fill the variable symbol to {{ variable_symbol }}.
+              {% endblocktrans %}
+            </div>
+
+            <p>
+              <strong>{% trans "Account name:" %}</strong> SPy<br>
+              <strong>{% trans "Bank name:" %}</strong> Fio banka, a. s., pobočka zahraničnej banky<br>
+              <strong>{% trans "Bank account EUR (SK):" %}</strong> 2300870269/8330<br>
+              <strong>{% trans "Bank account EUR (CZ):" %}</strong> 2300870269/2010<br>
+              <strong>{% trans "Bank account EUR (IBAN / international):" %}</strong> SK48 8330 0000 0023 0087 0269<br>
+              <strong>{% trans "SWIFT/BIC:" %}</strong> FIOZSKBAXXX
+            </p>
+
+            <h3>{% trans "Pay by PayPal" %}</h3>
+            <div class="alert alert-warning" role="alert">
+              {% trans "<strong>Warning!</strong> We charge additional 2% for PayPal payments." %}
+            </div>
+
+            <a href="{% url 'konfera_payments:paypal_button_url' order.uuid %}" class="btn btn-success">
+                {% trans "Pay by PayPal" %}
+            </a>
+          </div>
+        </div>
+        <hr>
+        <div class="row">
```

Comment:
I have a suggestion for this template. Lets use `{% extends 'konfera/order_details.html' %}` and in konfera/order_details.html add new block order_details_header (or something similar) and in this template instead of `{% block content %}` use `{% block order_details_header %}` this way you dont need all the code below, and if anything changes in konfera template will change here automatically as well. (additional benefit would be we can add setting for register redirect (default order_details), that can be set in local_settings to point here, without making konfera dependend on payments).